_staff_members/ariane_alves.md

```diff
@@ -1,6 +1,11 @@
 ---
 name: Ariane Alves
 city: São Paulo-SP
-image_path: /images/team/ariane_alves.jpeg
+image_path: /images/team/ariane_alves.png
+badged_image_path: /images/team/badged_team/ariane_alves.png
 twitter: ArianeCml
+  - 01
+  - 02
+  - 03
+  - 04
```

Comment:
Pegando a Ariane de exemplo, mas já notei a mesma situação em outros casos. Precisa localizar e ajustar todos.

**A pessoa com medalha não está aparecendo na página de medalhas**. 
A Ariane e a Camila ganharam medalha de divulgadoras, mas não aparece na seção. 

Ajuste: no arquivo [ariane_alves.md](https://raw.githubusercontent.com/okfn-brasil/embaixadoras/Atualiza-medalhas/_staff_members/ariane_alves.md) está faltando o título do campo `badges:` antes da lista de códigos. Em comparação o [camila_da_cruz.md](https://raw.githubusercontent.com/okfn-brasil/embaixadoras/Atualiza-medalhas/_staff_members/camila_da_cruz.md) está certo.
![image](https://github.com/okfn-brasil/embaixadoras/assets/44185775/830b4cbb-8ae9-456d-a853-99a24c9c8eba)
  
```suggestion
badges:
  - 01
  - 02
  - 03
  - 04
```

Reply:
Não ajuda muito pq você vai ter que conferir de qualquer forma, mas pelo visto tem 45 arquivos de imagens no diretório `badged_team` e se for usar o vscode para pesquisar quantos arquivos de `_staff_members` tem texto no formato certo do campo, são 39. 

Então, me parece, que são pelo menos 6 casos errados. 

![image](https://github.com/okfn-brasil/embaixadoras/assets/44185775/7d1e4844-c574-4c30-ab78-e77631ab16b6)
![image](https://github.com/okfn-brasil/embaixadoras/assets/44185775/4272ac5d-6ffe-4b09-b1b8-243006c4cf82)

Reply:
feito tb